_layouts/default.html

```diff
@@ -5,29 +5,6 @@
 
  {% include head.html %}
 
```

Comment:
Looks like the theme includes this video.js player as an alternative to the browser's default player: http://idratherbewriting.com/documentation-theme-jekyll/mydoc/mydoc_video_embeds.html

We will eventually embed or stream video into the docs, but I'm fine removing this for now if I can get your help with the right approach later.